stackle-app/src/app/secure/profile/profile.component.html

```diff
@@ -1,3 +1,41 @@
-<p>
-  profile works!
-</p>
+<div class="container">
```

Comment:
We can improve the profile with some more details like number of posts he has created. No of replies or comments he has made etc.

Reply:
I added three fields called posts, likes and comments.